tests/system/test_backup_api.py

```diff
@@ -289,6 +363,36 @@ def test_backup_create_w_invalid_version_time_future(
         op.result()  # blocks indefinitely
 
 
+def test_copy_backup_create_w_invalid_expire_time(
```

Comment:
nit: prefer to include this case in `test_copy_backup_workflow` in order to reuse the created source backup and minimize the number of created backups.